src/test/kotlin/uk/gov/dluhc/printapi/database/repository/PrintDetailsRepositoryTest.kt

```diff
@@ -81,6 +81,22 @@ internal class PrintDetailsRepositoryTest : IntegrationTest() {
             // Then
             assertThat(results).isEmpty()
         }
+
+        @Test
+        fun `should find eligible print requests`() {
+            // Given
+            printDetailsRepository.save(buildPrintDetails(batchId = null))
```

Comment:
As this test relies on these print details being `Status.PENDING_ASSIGNMENT_TO_BATCH` - can this status be explicitly set in the test set up?

Reply:
Done!